scripts/SANS/sans/gui_logic/sans_data_processor_gui_algorithm.py

```diff
@@ -45,7 +45,9 @@ def create_option_column_properties():
                                   property_type=float),
              algorithm_list_entry(column_name="",
                                   algorithm_property="EventSlices",
-                                  description='The event slices to take',
+                                  description='The event slices to reduce. The format is the same as for the event slices'
+                                              ' box in settings however if a comma separated list is given '
```

Comment:
if I am being fussy then you should have a comma before however. but it doesn't matter